src/App.js

```diff
@@ -1,25 +1,159 @@
+  const [message, setMessage] = useState("Deposit to play!");
+  const [balance, setBalance] = useState(100);
+  const [bet, setBet] = useState(credits[0]);
+  const [spins, setSpins] = useState(0);
+  const [wins, setWins] = useState(0);
+  const [losses, setLosses] = useState(0);
+
+  // Spins the reels
+  const spinReels = () => {
+    if (bet > balance) {
+      setMessage("Insufficient balance to spin!");
+      return;
+    }
+
+    setIsSpinning(true);
+    setMessage("");
+    setBalance(balance - bet);
+    setSpins(spins + 1);
+
+    let spinCount = 0;
+    const spinInterval = setInterval(() => {
+      setReels(
+        Array(3)
+          .fill(null)
+          .map(() =>
+            Array(3)
+              .fill(null)
+              .map(() => getRandomSymbol())
+          )
+      );
+      spinCount++;
+      if (spinCount > 10) {
+        clearInterval(spinInterval);
+        finalizeSpin();
+      }
+    }, 100);
+  };
+
+  // Checks if the player won
+  const checkWin = (finalReels) => {
+    const winningLines = [
+      // Horizontal lines
+      [finalReels[0][0], finalReels[1][0], finalReels[2][0]],
+      [finalReels[0][1], finalReels[1][1], finalReels[2][1]],
+      [finalReels[0][2], finalReels[1][2], finalReels[2][2]],
+      // Diagonal lines
+      [finalReels[0][0], finalReels[1][1], finalReels[2][2]],
+      [finalReels[0][2], finalReels[1][1], finalReels[2][0]],
+    ];
+
+    for (let line of winningLines) {
+      if (line[0] === line[1] && line[1] === line[2]) {
+        return true;
+      }
+    }
+    return false;
+  };
+
+  // Finalizes the spin and checks if the player won
+  const finalizeSpin = () => {
+    const finalReels = Array(3)
+      .fill(null)
+      .map(() =>
+        Array(3)
+          .fill(null)
+          .map(() => getRandomSymbol())
+      );
+    setReels(finalReels);
+    setIsSpinning(false);
+
+    if (checkWin(finalReels)) {
+      const winnings = bet * 10;
+      setBalance(balance + winnings);
+      setWins(wins + 1);
+      setMessage(`🎉 You won $${winnings}! 🎉`);
+    } else {
+      setLosses(losses + 1);
+      setMessage("Oops, Try again! 😢");
+    }
+  };
+
+  const increaseBet = () => {
+    if (bet < balance) {
+      setBet(credits[credits.indexOf(bet) + 1]);
+    }
+  };
+
+  const decreaseBet = () => {
+    if (bet > 1) {
+      setBet(credits[credits.indexOf(bet) - 1]);
+    }
+  };
 
-function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="slot-machine">
+      <h1 className="title">🥥 Coco Spins 🥥</h1>
+      <div className="reels">
```

Comment:
modularize these and move them into their respective components. You already have the right classname. Instead of creating className to separate it, you can just isolate it by moving to its own component and then the styling will be easier to target.